src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="temp"
+        title="OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317 (For the US)"
+      >
+        This is the endpoint your application telemetry will be exported to. Export data to New Relic's OTLP endpoint. If you're in the EU, use this endpoint:
+
+        OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.eu01.nr-data.net:4317
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="service-name"
+        title="OTEL_SERVICE_NAME=getting-started-java"
+      >
+        This is the name that associates your application telemetry with an entity in New Relic. In this tutorial we're calling it `getting-started-java`, but you could call it anything you want.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="instance-id"
+        title="OTEL_RESOURCE_ATTRIBUTES=service.instance.id=INSERT_YOUR_OWN_ID_HERE"
+      >
+        Replace `INSERT_YOUR_OWN_ID_HERE` with a unique instance id. For example, you could use `1234`. For more details, see OpenTelemetry's [Resource Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md#service).
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="length-limit"
+        title="OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT=4095"
+      >
+        Setting a limit on the maximum attribute value length will prevent span data from getting dropped if it exceeds the [New Relic limit](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-attributes/).
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="temp2"
+        title="OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=DELTA"
+      >
+        New Relic supports metrics in delta temporality instead of the default of cumulative.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="bucket-hist"
+        title="OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION=EXPONENTIAL_BUCKET_HISTOGRAM"
+      >
+        (Recommended) Histogram aggregation: Use exponential histogram instead of default explicit bucket histogram for better data compression.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="logs-otlp"
+        title="OTEL_LOGS_EXPORTER=otlp"
+      >
+        Enable log export over OTLP. Log export is disabled by default.
+      </Collapser>
+      <Collapser
+        className="freq-link"
+        id="gzip"
+        title="OTEL_EXPORTER_OTLP_COMPRESSION=gzip"
+      >
+        (Recommended) Setting gzip compression allows you to maximize the amount of data you can send per request. The New Relic maximum allowed payload size is 1MB (10^6 bytes). Gzip compression has good performance and lowers data egress.
```

Comment:
Are you okay saying just 'Our' instead of 'The New Relic'? Like 'Our maximum allowed...'

Reply:
@zuluecho9, I changed this to "Our" in both instances. Thanks!